platforms/bukkit/build.gradle

```diff
@@ -19,12 +20,17 @@ repositories
     {   // For Bukkit API, updated by Spigot
 		url = "https://hub.spigotmc.org/nexus/content/repositories/snapshots/"
     }
+    maven 
+    {   // Worldedit API
+   		url "https://maven.sk89q.com/repo/" 
+   	}
 }
 
 dependencies
 {
     compile 'org.bukkit:bukkit:' + project.ext.bukkitVersion
     compile 'org.spigotmc:spigot:' + project.ext.bukkitVersion
+    compileOnly 'com.sk89q.worldedit:worldedit-bukkit:' + project.ext.worldeditVersion
```

Comment:
Would be interested to know what happens if you import and use WE classes at mod init, and then run the client without WE. It should crash with something like a NoClassDefFoundError / ClassNotFoundException / MethodNotFoundException. It think it doesn't currently crash because the OTG classes that use the WE dependencies are never loaded if WE isnt installed, OTG commands never reach that code, which is nice.